doc/source/tune/tutorials/tune-output.rst

```diff
@@ -210,13 +210,20 @@ You can do this in the trainable, as shown below:
             for step in range(100):
                 library.log_model(...)
                 library.log(results, step=step)
+
+                # You can also just write to a file directly.
+                with open(f"./artifact_{step}.txt", "w") as f:
```

Comment:
can you note something about the local dir being a trial directory?

Reply:
I have a few sentences about this right below, but will also add a bit of info to the inline comment.